coalib/parsing/DefaultArgParser.py

```diff
@@ -251,6 +251,17 @@ def default_arg_parser(formatter_class=None):
         ' additional arguments. If used without arguments, all bears specified'
         ' with --bears will be debugged (even implicit dependency bears).')
 
+    outputs_group.add_argument(
+        '--profile', nargs='?', const=True,
```

Comment:
you need to create an arg group for `--profile` and `--debug` and set that group to only allow one of them.
See argparse docs

Reply:
This is mandatory @Makman2 .  More at https://github.com/coala/coala/pull/5599#discussion_r207759623